.github/workflows/cpp.yml

```diff
@@ -53,7 +53,7 @@ jobs:
     name: ${{ matrix.title }}
     runs-on: ubuntu-latest
     if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
-    timeout-minutes: 45
```

Comment:
Asan job runs about 44m in a recent test https://github.com/apache/arrow/runs/4100836200
Sometimes it timeout.